WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -1124,36 +1124,33 @@ def finalize(self):
     # find if there are pending failover requests
     requests = self.__getRequestFiles()
     outputDataRequest = self.failoverTransfer.getRequest()
+    pendingRequests = 'Pending Requests'
```

Comment:
I am tempted to have a `JobMinorStates` module where this, and other "common" minor states are defined. Like for the `JobStates` module.

Reply:
We could put them in the same file `JobStatus.py`?

Reply:
I think we should have another file.

Reply:
Done